### ISSUE TITLE
[Bug]: Time spent inside a raid has become around 52 years 357 days after actually only spending around 15 to 20min

### ISSUE THREAD

**Issue by BalkrishanSingh** [MelvorIdle/melvoridle.github.io#3108]
### Describe the bug

The time spent in a raid became near 53 years. 
not certain it will be replicable easily since alot of the people on discord didnt have this issue  but i found another reddit post describing the same issue
https://www.reddit.com/r/MelvorIdle/comments/zezhy5/fell_asleep_with_the_goblin_raid_going_came_back/
If it is not possible to post a fix to this anytime soon in a update. could you tell me a command or something to reset my goblin raid statistics. I dont care if the raid points and purchases are gone.
another thing of note is i am on the demo and haven't yet actually gone the full game. so maybe thats something i dont have in common with people not experiencing the bug.

### Reproduction Steps

1. Start a goblin raid
2. Play for a while
3. End the raid

### Expected behaviour

it should show the time i actually spent which should be between 15 to 20min.

### Save Link

https://melvoridle.com/save/Ua5wNyQw

### Screenshots

<img width="677" alt="Screen Shot 2022-12-21 at 00 07 42" src="https://user-images.githubusercontent.com/93041640/208741297-5eae3115-5678-4708-875e-bc518332c5d8.png">
<img width="699" alt="Screen Shot 2022-12-21 at 00 04 30" src="https://user-images.githubusercontent.com/93041640/208740982-a15a8c47-98ee-4a73-af39-807d2d66ee09.png">

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [X] Safari
- [ ] Mobile App (iOS)
- [X] Mobile App (Android)
- [ ] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.1.1

### Game Subversion

?4970

### Are you using any mods?

No

**Comment by xanderput** [comment]
Since I know a bit more about this I'll elaborate here.

The start time of a raid is represented as ms since epoch. 
When encoding this value for saving, it gets encoded as UINT32 which as a max of ~4.3b.
This corresponds to 19th of February 1970 so it basically always overflows.

**Comment by Just-a-Cactus** [comment]
Yes, I caught this bug too. For me it's happened when I **start** a golbin raid **before** global "trone" update and **finish** raid **after update**... Before trone update all worked good. I'm not started new raid after that happened so I don't know, is it happen if you start/finish at same trone version
<img width="609" alt="Screenshot 2022-12-22 at 10 44 33" src="https://user-images.githubusercontent.com/43491424/209094261-485a039b-fae2-40b9-b774-19ea30133210.png">
